packages/back-end/src/controllers/ideas.ts

```diff
@@ -41,13 +39,16 @@ export async function getIdeas(
 }
 
 export async function getEstimatedImpact(
-  req: AuthRequest<{ metric: string; segment?: string }>,
+  req: AuthRequest<{ metric: string; segment?: string; ideaId?: string }>,
   res: Response
 ) {
   req.checkPermissions("createIdeas", "");
-  req.checkPermissions("runQueries", "");
 
-  const { metric, segment } = req.body;
+  const { metric, segment, ideaId } = req.body;
+
+  const idea = await getIdeaById(ideaId || "");
+
+  req.checkPermissions("createIdeas", idea?.project || "");
```

Comment:
I think this is a typo.  Should be `runQueries`

Reply:
DOH. Updated